### PR TITLE
DHFPROD-7934: Navigate to Graph Settings

### PR DESCRIPTION
**Pull request by xnikhil08**
### Description
Just a dummy graph settings modal UI. Saving the modal will come in future stories.

#### Checklist: 
```diff
- Note: do not change the below
```

-  ##### Owner:

- [x] JIRA_ID included in all the commit messages
- [x] PR title is in the format JIRA_ID:Title
- [x] Rebase the branch with upstream
- [x] Squashed all commits into a single commit
- [x] Code passes ESLint tests
- [x] Added Tests
- [x] Run UI tests
  

- ##### Reviewer:

- [ ] Reviewed Tests

### CODE REVIEW

**Review by federicoRufrancosMarklogic — CHANGES_REQUESTED** [review]
Code looks good, I found some comment out code, are they necessary?

On testing functionality I don't see filter over Entity Type. Here is an screenshot
image

And here is how it looks on UX doc:
image

Is color selection part of this story? Because I don't see anything detailed on the story. Probably is part of another task

File: marklogic-data-hub-central/ui/e2e/cypress/support/pages/browse.tsx [hunk 1]
```diff
@@ -521,10 +521,18 @@ class BrowsePage {
...
+  getExploreSettingsMenuIcon() {
+    return cy.get("[aria-label=explore-settings-menu]");
+  }
+
   getManageQueriesButton() {
-    return cy.get("#manage-queries-button");
+    return cy.get("[aria-label=manageQueries]");
   }
 
+  // getManageQueriesButton() {
```
[the remark this answers is not in the document]
Comment (xnikhil08) [reply marklogic-data-hub-central/ui/e2e/cypress/support/pages/browse.tsx:536]: Filter, color and Icon have their own separate stories. Not in scope here.

**Review by Sanjeevani19 — COMMENTED** [review]
@xnikhil08 Functionality looks good !!

-  Still see some commented code, Is there a reason for not removing them from the code base ?

- Also there is one new e2e intermittent failure related to manage queries modal which is happening only on this PR, I see it on runs 6 and 8, Please look into fixing it

 **verify manage queries modal visibility and removing query scenario on the detail page**

**Comment by xnikhil08** [comment]
> @xnikhil08 Functionality looks good !!
> 
> * Still see some commented code, Is there a reason for not removing them from the code base ?
> * Also there is one new e2e intermittent failure related to manage queries modal which is happening only on this PR, I see it on runs 6 and 8, Please look into fixing it
> 
> **verify manage queries modal visibility and removing query scenario on the detail page**

@Sanjeevani19 - Fixed the failing test. Another review ?

**Comment by federicoRufrancosMarklogic** [comment]
* Entity types should be listed in alphabetical order by default
* On the story said: "There's a filter for entity types and the ability to sort entity types", so should we remove "filter for entity types"? as you mention is part of different story
* Entity types header should be aligned with column content
image
* Header column font should be stronger

**Comment by xnikhil08** [comment]
Closing it for now, since it has been de-prioritized in the 5.7.

**Pull request closed without merging**